Firestore/core/src/firebase/firestore/model/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# firebase_firestore_util is the interface of this module. The rest of the
```

Comment:
This is `firebase_firestore_model` :-).

There shouldn't be any multi-platform shenanigans here so I think you can just remove this comment.